### ISSUE TITLE
TODO lists

### ISSUE THREAD

**Issue by haozixu** [haozixu/MTCNN-Keras#1]
1. fix precision and f1 metric
2. add evaluation and run code
3. bugs at pnet softmax; remove the first column
4. improve performance
